lib/daru/dataframe.rb

```diff
@@ -1839,23 +1947,16 @@ def prepare_vector_for_insert name, vector
       end
     end
 
-    def insert_or_modify_row name, vector
-      raise NotImplementedError, "Still can't insert rows in multi index dataframes" \
-        if index.is_a?(MultiIndex)
+    def insert_or_modify_row indexes, vector
+      vector = preprocess_vector vector
 
-      name = name[0]
-      vec = Vector.coerce(vector, name: coerce_name(name), index: @vectors)
+      raise SizeError, 'Vector length should match row length' if
+        vector.size != @vectors.size
 
```

Comment:
I need to declare `SizeError` exception. Where shall I do it? Or should I use `ArgumentError` instead?